pkgs/applications/audio/fverb/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, faust2lv2 }:
+stdenv.mkDerivation rec {
```

```suggestion

stdenv.mkDerivation rec {
```
empty line for clarity